### ISSUE TITLE
CocoaPods

### ISSUE THREAD

**Issue by luciocouto** [card-io/card.io-iOS-SDK#192]
I have this app in app and it works. I tried to change to CocoaPods and only in cocoaPods, it's crashes when i read a card. no log at all, just crash. but only when it is on cocoapods... any clue?

**Comment by bluk** [comment]
@luciocouto Are there any other libraries you included via CocoaPods? Also, are you using the workspace generated by CocoaPods?

**Comment by luciocouto** [comment]
Yes, there is and yes i am! In cocoapods, crashs just when it reads the card, if there's no card to read and the user cancel the action, it works fine.

**Comment by bluk** [comment]
Without any logs, this is difficult to debug. I would double check that your build settings are still correct for your app target (e.g. have the `$(inherited)` value in the target's build settings where appropriate as well).
